kotlin/workflow-runtime/src/main/java/com/squareup/workflow/internal/WorkerChildNode.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.workflow.internal
+
+import com.squareup.workflow.Worker
+import com.squareup.workflow.WorkflowAction
+import com.squareup.workflow.internal.InlineLinkedList.InlineListNode
+import kotlinx.coroutines.channels.ReceiveChannel
+
+/**
+ * TODO kdoc
+ *
+ *
+ * Holds the channel representing the outputs of a worker, as well as a tombstone flag that is
+ * true after the worker has finished and we've reported that fact to the workflow. This is to
+ * prevent the workflow from entering an infinite loop of getting `Finished` events if it
+ * continues to listen to the worker after it finishes.
+ *
+ * @param channel TODO
+ * @param tombstone TODO
+ */
+/* ktlint-disable parameter-list-wrapping */
```

Comment:
You're not wrapping the param list.

Reply:
Not what ktlint thinks – it fails without this. It's because of the weird indentation due to the generics.